core/modules/contact/contact.install

```diff
@@ -28,6 +28,46 @@ function contact_install() {
   }
 }
 
+/**
+ * Implements hook_enable().
+ */
+function contact_enable() {
```

Comment:
I think this should be in `contact_install()` rather than in `_enable()`, as if you made customizations to the menu items, you wouldn't want them modified when temporarily disabling/enabling contact module.